ripe/atlas/sagan/helpers/compatibility.py

```diff
@@ -22,3 +22,38 @@
 except NameError:
     string = str  # Python3
 
```

Comment:
Althouth the rest of the metaclasses solution is straight forward the below part is blowing my mind :) I have hard time to understand...it's like magic on top of magic.